### PR TITLE
Remove wrong event handlers

### PR DESCRIPTION
**Pull request by adhami3310**
also deprecate default annotations for strings (which means there will be good amount of fixes to event handlers in the library soon)

### CODE REVIEW

File: reflex/components/radix/themes/components/text_area.py [hunk 1]
```diff
@@ -83,19 +83,19 @@ class TextArea(RadixThemesComponent, elements.Textarea):
     wrap: Var[str]
 
     # Fired when the value of the textarea changes.
-    on_change: EventHandler[lambda e0: [e0.target.value]]
+    on_change: EventHandler[input_event]
```
Comment (masenf) [inline reflex/components/radix/themes/components/text_area.py:86]: if this class inherits from `elements.Textarea`, do we need these definitions here? are they for autodoc purposes?

Comment (masenf) [reply reflex/components/radix/themes/components/text_area.py:86]: Soon TextField will inherit from `elements.Input`, so similar question there.

**Review by masenf — APPROVED** [review]
Probably followup work, but the drawer content component accepts `on_pointer_down_outside` and `on_open_auto_focus` events.

Looks like the root also exposes `on_close`.

The docs for vaul are a bit out of date with the code; we probably should go through and clean these up.

**Comment by adhami3310** [comment]
> the drawer content component accepts on_pointer_down_outside and on_open_auto_focus events

it technically does, but also it's not documented in vaul, so we kinda agreed to leave it out

File: reflex/components/radix/themes/components/text_area.py (hunk 1 above)
Comment (adhami3310) [reply reflex/components/radix/themes/components/text_area.py]: removed unnecessary ones! should work fine as PYI file remained the same

**Review by masenf — APPROVED** [review]

**Pull request merged**